spec/requests/users_request_spec.rb

```diff
@@ -14,37 +14,13 @@
     it 'renders correct template' do
       expect(response).to_not render_template('show')
     end
-
-    it 'check for placeholder text' do
-      expect(response.body).to include('All Users')
-    end
-
-    it 'check for placeholder text' do
-      expect(response.body).to include('Here is a list of all users')
-    end
   end
 
   describe 'GET /show' do
     before(:example) { get('/users/365') }
 
```

Comment:
- [ ] You still have one failing test. Kindly look into it.
![Screenshot from 2021-12-18 23-12-28](https://user-images.githubusercontent.com/65501535/146654507-d61b9957-6ac4-4928-92d1-3ef300d59e71.png)

Reply:
As seen earlier in the first review, this same test was failing. But since it is passing on your computer, I would like to think that we just have different config setups for this case.
> NOTE: Next reviewer, I suggest you approve this CRR. 😄

Reply:
Thank you a lot. I really appreciate it. 💪 👍